mlflow/azureml/__init__.py

```diff
@@ -31,6 +31,16 @@ def build_image(model_path, workspace, run_id=None, image_name=None, model_name=
     The resulting image can be deployed as a web service to Azure Container Instances (ACI) or
     Azure Kubernetes Service (AKS).
 
+    The resulting Azure ML ContainerImage will contain a webserver that accepts the following
+    data formats as input:
+
+    - JSON-serialized Pandas Dataframes in the `split` orientation. For example,
+      `data = pandas_df.to_json(orient='split')`. This format is specified using a `Content-Type`
+      request header value of `application/json.pandas.split.oriented`.
```

Comment:
Here, this should just be `application/json`, right?

Reply:
Good catch! Changed the docs to `application/json` for AzureML.